frontend/src/styles/components/_Codes.sass

```diff
@@ -101,3 +100,8 @@
     @include flexbox()
     @include justify-content(center)
     padding-top: $spacing-layout-m
+
+  .list-footer
+      margin-left: auto
+      margin-right: auto
+      margin-top: 1em
```

Comment:
To stop GitHub from warning about this, you should enable 'Ensure every saved file ends with a line break' setting in your IDE.

Reply:
Done :)